enterprise_access/apps/api_client/tests/test_utils.py

```diff
@@ -15,3 +22,99 @@ def __init__(self, json_data, status_code):
+        self.mock_license_activation_key = self.faker.uuid4()
+        self.mock_auto_apply_uuid = self.faker.uuid4()
+        self.mock_subscription_plan_uuid = self.faker.uuid4()
+        self.mock_customer_agreement_uuid = self.faker.uuid4()
+
+        self.mock_subscription_plan = {
+            "title": "mock_title",
+            "uuid": self.mock_subscription_plan_uuid,
+            "start_date": _days_from_now(-50, DATE_FORMAT_ISO_8601),
+            "expiration_date": _days_from_now(50, DATE_FORMAT_ISO_8601),
+            "enterprise_customer_uuid": self.mock_enterprise_customer_uuid,
+            "enterprise_catalog_uuid": self.mock_enterprise_catalog_uuid,
+            "is_active": True,
+            "is_current": True,
+            "is_revocation_cap_enabled": False,
+            "days_until_expiration": 50,
+            "days_until_expiration_including_renewals": 50,
+            "is_locked_for_renewal_processing": False,
+            "should_auto_apply_licenses": False,
+            "created": _days_from_now(-60, DATE_FORMAT_ISO_8601)
+        }
+        self.mock_customer_agreement = {
+            "uuid": self.mock_customer_agreement_uuid,
+            "enterprise_customer_uuid": self.mock_enterprise_customer_uuid,
+            "enterprise_customer_slug": self.mock_enterprise_customer_slug,
+            "default_enterprise_catalog_uuid": self.mock_enterprise_catalog_uuid,
+            "disable_expiration_notifications": False,
+            "net_days_until_expiration": 50,
+            "subscription_for_auto_applied_licenses": self.mock_subscription_plan_uuid,
+            "available_subscription_catalogs": [
+                self.mock_enterprise_catalog_uuid,
+            ],
+            "enable_auto_applied_subscriptions_with_universal_link": False,
+            "has_custom_license_expiration_messaging": False,
+            "modal_header_text": None,
+            "expired_subscription_modal_messaging": None,
+            "button_label_in_modal": None,
+            "url_for_button_in_modal": None
+        }
+        self.mock_subscription_license = {
+            "uuid": self.mock_learner_license_activation_uuid,
+            "status": LicenseStatuses.ACTIVATED,
+            "user_email": self.mock_user_email,
+            "activation_date": _days_from_now(-10, DATE_FORMAT_ISO_8601_MS),
+            "last_remind_date": _days_from_now(-5, DATE_FORMAT_ISO_8601_MS),
+            "subscription_plan_uuid": self.mock_subscription_plan_uuid,
+            "revoked_date": None,
+            "activation_key": self.mock_license_activation_key,
+            "subscription_plan": self.mock_subscription_plan
+        }
+        self.mock_learner_license_activation_response = {
+            **self.mock_subscription_license,
+            "uuid": self.mock_learner_license_activation_uuid,
+            "status": LicenseStatuses.ACTIVATED,
+            "user_email": self.mock_user_email,
```

Comment:
nit: `user_email` is the same as `self.mock_subscription_license`, and is already spread via line 109 above.

nit x2: technically the same for `activation_date`, `last_remind_date`, and `status` too. It seems the only thing really different is the `uuid`?